inc/class-wpseo-primary-term.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * @package WPSEO
+ */
+
+/**
+ * Represents a post's primary term
+ */
+class WPSEO_Primary_Term {
+
+	/**
+	 * @var string
+	 */
+	protected $taxonomy_name;
+
+	/**
+	 * @var int
+	 */
+	protected $post_ID;
+
+	/**
+	 * The taxonomy this term is part of
+	 *
+	 * @param string $taxonomy_name Taxonomy name for the term.
+	 * @param int    $post_ID       Post ID for the term.
+	 */
+	public function __construct( $taxonomy_name, $post_ID ) {
+		$this->taxonomy_name = $taxonomy_name;
+		$this->post_ID = $post_ID;
+	}
+
+	/**
+	 * Returns the primary term ID
+	 *
+	 * @return int
+	 */
+	public function get_primary_term() {
+		$primary_term = get_post_meta( $this->post_ID, WPSEO_Meta::$meta_prefix . 'primary_' . $this->taxonomy_name, true );
+
+		// By default the first term (sorted by ID) is the primary term.
```

Comment:
@atimmer and I have decided to keep the code as it is, for the following reason:
In the current code there is only 1 return and the function (and its goal) is easy to follow. With the suggestion, there will be 3 returns which can be quite confusing.